src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2690,7 +2690,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>This warning occurs if the assembly attributes AssemblyKeyFileAttribute or AssemblyKeyNameAttribute found in source conflict with the /keyfile or /keycontainer command line option or key file name or key container specified in the Project Properties.</value>
   </data>
   <data name="ERR_BadCompatMode" xml:space="preserve">
-    <value>Invalid option '{0}' for /langversion; must be ISO-1, ISO-2, Default or an integer in range 1 to 7.</value>
+    <value>Invalid option '{0}' for /langversion; must be ISO-1, ISO-2, Default, Latest or a valid version in range 1 to 7.1.</value>
```

Comment:
Wouldn't it be too long? Consider in a couple of point releases:
Invalid option '{0}' for /langversion; must be ISO-1, ISO-2, Default, Latest, 1, 2, 3, 4, 5, 6, 7, 7,1, 7.2, 8.
I'd claim that people calling csc from command line already know what they're doing (and they don't need to know which C# versions were shipped), VS UI or first time users who would normally use VS project dialogs to enumerate available versions.


---
In reply to: [106479809](https://github.com/dotnet/roslyn/pull/17894#discussion_r106479809) [](ancestors = 106479809)